Process.h

```diff
@@ -84,6 +112,12 @@ typedef struct Process_ {
    /* Foreground group identifier of the controlling terminal */
    int tpgid;
 
+   /* This is a kernel (helper) task */
+   bool isKernelThread;
+
+   /* This is a userland thread / LWP */
+   bool isUserThread;
```

Comment:
In many (if not all) other places this field was 'isUserlandThread' including in the comment above - could we stick with that convention?  There are functions named isUserlandThread() and fields in structures in ProcessList.h and Settings.h that are also named with the 'userland' convention.

Reply:
Will update the patch series accordingly to use `isUserlandThread`.